src/service/viewer-impl.js

```diff
@@ -419,6 +417,12 @@ export class Viewer {
    * @return {boolean}
    */
   hasCapability(name) {
+    // TODO(lannka, #4457): ideally, we should check if viewer has the capability
+    // of CID storage, rather than if it is embedded.
+    if (name == 'cid') {
+      return isIframed(this.win);
```

Comment:
We need to at least reach out to webview folk to confirm. I believe this is correct, however. We only do this for iframes.

RE: viewer cid. Do the viewers already set this capability? Does it make sense to just switch to that?